package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gdax-trading-toolkit",
-  "version": "0.1.18",
+  "version": "0.1.19",
```

Comment:
Let us manage versioning; we can then batch a bunch of PRs together in a release.